src/components/SharedComponents/Accordion.style.tsx

```diff
@@ -23,13 +25,21 @@ export const StyledAccordionSummary = styled(AccordionSummary)`
 
 export const MarkdownBody = styled(MarkdownContent)`
   color: ${COLOR_MAP.GRAY_BODY_COPY};
+  line-height: 1.4;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 22px;
 
   p {
-    font-size: 1rem;
     line-height: 1.4;
+    font-family: Roboto;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
```

Comment:
line-height is declared here twice now (`line-height: 1.4` a few lines up). which is the correct one?

Reply:
1.6 is closest to what we have in Figma, updating